src/core/Track.cpp

```diff
@@ -63,7 +63,7 @@ Track::Track( Type type, TrackContainer * tc ) :
 	m_name(),                       /*!< The track's name */
 	m_mutedModel( false, this, tr( "Mute" ) ), /*!< For controlling track muting */
 	m_soloModel( false, this, tr( "Solo" ) ), /*!< For controlling track soloing */
-	m_simpleSerializingMode( false ),
+	m_presetMode(false),
```

Comment:
Per [Cpp Core Guidelines](https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#c48-prefer-default-member-initializers-to-member-initializers-in-constructors-for-constant-initializers), this should go into the header.

Reply:
Adjusted with commit https://github.com/LMMS/lmms/pull/7445/commits/a1c276debf59d278182f0ebd9658e43c70bca015.